test/test_core.py

```diff
@@ -51,6 +51,22 @@ def decorated(self, *args, **kwargs):
   return decorated
 
 
+def no_modularize_instance(note):
+  assert not callable(note)
+
+  def decorator(f):
+    assert callable(f)
+
+    @wraps(f)
+    def decorated(self, *args, **kwargs):
+      if self.get_setting('MODULARIZE') == 'instance' or self.get_setting('WASM_ESM_INTEGRATION'):
```

Comment:
Should the name refer to both properties checked here? `no_modularize_instance_or_esm`?

Reply:
`WASM_ESM_INTEGRATION` automatically implies `MODULARIZE == instance` so it kind of redundant.